content/guides/embed/ui-elements/ui-elements-design-tokens.md

```diff
@@ -0,0 +1,348 @@
+---
+rank: 12
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+  - embed/ui-elements/theming-styling
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+---
+
+# UI Elements design tokens
+
+This table shows all the design tokens you can use for [theming and styling][ts] purposes. Provided default values and descriptions give clarity and guidance for implementation.
+
+<!--alex ignore -->
+
+## Border colors
```

Comment:
Add message: 
You can pass any CSS color value, like hexadecimal colors, RGB, cross-browser color names, etc.